distribution/src/bin/openfire.sh

```diff
@@ -138,6 +138,11 @@ case $arguments in
 esac
 done
 
+# Java security config
+OPENFIRE_OPTS="${OPENFIRE_OPTS} -Djava.security.properties=${OPENFIRE_HOME}/resources/security/java.security"
```

Comment:
Does this _add_ to the JVM-provided security properties, or does it replace them (I'm not immediately finding heaps of documentation on this property)? In the latter case, I worry about loosing other, possibly important, configuration.

Reply:
This adds to the existing opts by including them at the start `${OPENFIRE_OPTS} `